src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/visualeditordialogs/VisualEditorEditTemplateDialog.java

```diff
@@ -62,15 +61,15 @@ public ArticlePageObject clickGetInfoLink() {
   public void typeInParam(String paramName, String text) {
     waitForDialogVisible();
     if (isElementOnPage(TEMPLATE_PARAMS_BY)) {
-      WebElement targetParam = Elements.getElementByChildText(templateParams, PARAM_LABEL_BY, paramName);
+      WebElement targetParam =
+          Elements.getElementByChildText(templateParams, PARAM_LABEL_BY, paramName);
       WebElement targetParamInput = targetParam.findElement(PARAM_INPUT_BY);
       targetParamInput.sendKeys(text);
       waitForValueToBePresentInElementsAttributeByElement(targetParamInput, "value", text);
     } else {
       throw new NoSuchElementException("This template has no param.");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "This template has no param." 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)